### DISCUSSION TITLE
Error integrating GrapeJS with Vuexy Template (MUI + NextJs)

### DISCUSSION THREAD

**Discussion by lucasmuleyv** [GrapesJS/grapesjs#5566] [Q&A]
Hello everyone,

I'm currently experiencing an issue while attempting to integrate GrapeJS into the Vuexy Template, which utilizes MUI and NextJs. Additionally, I'm using the GrapeJS React package available at https://github.com/GrapesJS/react. The specific error I'm encountering is a TypeError: "Object.defineProperty called on non-object."

To diagnose the problem, I tried implementing GrapeJS in a basic NextJS setup, where it functioned without issues. This leads me to believe the problem might be associated with the Vuexy template, although I can't be certain. Has anyone else encountered a similar issue?

Any insights or advice would be greatly appreciated. Thank you in advance for your help!

Best regards,
Lucas M.

[package.json](https://github.com/GrapesJS/grapesjs/files/13714391/package.json)

Sandbox:
https://codesandbox.io/p/github/lucasmuleyv/example-vuexy-grapejs-javascript/master

**Comment by artf** [comment]
I believe without a reproducible demo you hardly get any kind of help.

**Reply by lucasmuleyv** [reply]
Sorry! Here an example: https://codesandbox.io/p/github/lucasmuleyv/example-vuexy-grapejs-javascript/master

**Reply by artf** [reply]
Unable to access the workspace
```
It is likely that you are not a member of the team that this project belongs to. To be able to access the web editor of this repository and edit the code, you need to be invited to the team.
```
